src/main/java/tictactoe/Board.java

```diff
@@ -31,16 +31,16 @@ private int getColumn(int position) {
         return position % size;
     }
 
-    public final MarksEnum[][] getContent() {
+    public MarksEnum[][] getContent() {
```

Comment:
I did the same mistake with my Ruby version. I assume that I still misunderstood the problematic then.

I understand that we want an interface that always return the same data-structure which is currently a double array. Imagine that I have to change the double array to a list, I will have to convert my list to a double array in order to respect the interface. 

Can't I consider the double array as my abstraction?